README.md

```diff
@@ -50,14 +50,20 @@ Polaris is built using Gradle with Java 21+ and Docker 27+.
 - `./gradlew assemble` - To skip tests.
 - `./gradlew test` - To run unit tests and integration tests.
 - `./gradlew runApp` - To run the Polaris server locally on localhost:8181. 
-  - The server starts with the in-memory mode, and it prints the auto-generated credentials to STDOUT in a message like this `realm: default-realm root principal credentials: <id>:<secret>`
-  - These credentials can be used as "Client ID" and "Client Secret" in OAuth2 requests (e.g. the `curl` command below).
-- `./regtests/run.sh` - To run regression tests or end-to-end tests in another terminal.
+- `./regtests/run_spark_sql.sh` - To connect from Spark SQL. Here are some example commands to run in the Spark SQL shell:
+```sql
+create database db1;
+show databases;
+create table db1.table1 (id int, name string);
+insert into db1.table1 values (1, 'a');
+select * from db1.table1;
+drop table db1.table1;
+```
 
+### More build and run options
```

Comment:
Adding a subtitle to separate the advanced build/run options from the basic ones.

Reply:
I think maybe we should separate out the spark sql command. Currently it is like

Build Stuff
```
- `./gradlew build` - To build and run tests. Make sure Docker is running, as the integration tests depend on it.
- `./gradlew assemble` - To skip tests.
- `./gradlew test` - To run unit tests and integration tests.
```

Start Service Locally
```
- `./gradlew runApp` - To run the Polaris server locally on localhost:8181. 
```


Quick Demo // Unrelated?
```
- `./regtests/run_spark_sql.sh` - To connect from Spark SQL. Here are some example commands to run in the Spark SQL shell:
```

Reply:
I thought this a lot of time actually, we want it to be more logical organized. But I didn't do this since it makes the page much longer. I still prefer a compacted front page.